build/generate-manifests.sh

```diff
@@ -37,6 +37,17 @@ else
   OLM_FEATURES="[\"Disconnected\",\"fips\"]"
   ARCHITECTURE_LABELS=$(generateArchitectureLabels amd64 s390x ppc64le arm64)
   OLM_SUBSCRIPTION_ANNOTATION="operators.openshift.io/valid-subscription: '[\"OpenShift Container Platform\", \"OpenShift Platform Plus\"]'"
+  OLM_FEATURE_DISCONNECTED="features.operators.openshift.io/disconnected: \"true\""
+  OLM_FEATURE_FIPS_COMPLIANT="features.operators.openshift.io/fips-compliant: \"true\""
+  OLM_FEATURE_PROXY_AWARE="features.operators.openshift.io/proxy-aware: \"false\""
+  OLM_FEATURE_TLS_PROFILES="features.operators.openshift.io/tls-profiles: \"false\""
+  OLM_FEATURE_TOKEN_AUTH_AWS="features.operators.openshift.io/token-auth-aws: \"false\""
+  OLM_FEATURE_TOKEN_AUTH_AZURE="features.operators.openshift.io/token-auth-azure: \"false\""
+  OLM_FEATURE_TOKEN_AUTH_GCP="features.operators.openshift.io/token-auth-gcp: \"false\""
+  OLM_FEATURE_CNF="features.operators.openshift.io/cnf: \"false\""
+  OLM_FEATURE_CNI="features.operators.openshift.io/cni: \"false\""
+  OLM_FEATURE_CSI="features.operators.openshift.io/csi: \"false\""
```

Comment:
To stay consistent, these variable names should end with `_ANNOTATION` just like the existing `OLM_SUBSCRIPTION_ANNOTATION` variable. 

Alternatively, instead of including the entire annotation, you could also just keep the annotation value, just like we do for `OLM_FEATURES`, which is used to set the value of the `operators.openshift.io/infrastructure-features` annotation.